lib/records/RecHttp.cc

```diff
@@ -649,9 +649,10 @@ HttpProxyPort::print(char *out, size_t n)
       zret += snprintf(out + zret, n - zret, ":%s=", OPT_PROTO_PREFIX);
     }
     bool sep_p = !need_colon_p;
-    for (int k = 0; k < SessionProtocolSet::MAX; ++k) {
+    for (unsigned k = 0; k < SessionProtocolSet::MAX; ++k) {
```

Comment:
Why this is changed to `unsigned`? It looks like `globalSessionProtocolNameRegistry.nameFor(k)` takes `int`.

Reply:
Because the index goes from 0 to positive numbers. I'm trying to be more signed compliant.

Reply:
Oh well, I'll tweak that back.